### ISSUE TITLE
Setting ```__debug``` in ```amortized_2pc.h``` is broken

### ISSUE THREAD

**Issue by joerowell** [emp-toolkit/emp-ag2pc#36]
If one sets ```__debug``` in ```amortized_2pc.h``` then the code doesn't compile.  This is because variables like ```value``` aren't defined in the class anymore.

**Comment by wangxiao1254** [comment]
Yup they are not embedded in the last bit of the mac. Will update it.
